nav-app/src/app/tabs/tabs.component.ts

```diff
@@ -54,6 +54,11 @@ export class TabsComponent implements AfterContentInit, AfterViewInit {
       icon: 'https://www.quantum.security/assets/img/home/xdr-heading-alternate.svg',
       name: 'Quantum XDR+SOC Rule Library',
       description: 'Layer showing coverage of the Quantum Security detection rule library.',
+    }, {
+        href: 'siem/azure-quantum-rules.json',
```

Comment:
It doesn't look like this file was committed.

Reply:
Looks like this is dependent on the upstream PR, but the submodule will need to be updated once that's merged.